setup.py

```diff
@@ -84,7 +84,7 @@ def __str__(self):
 if sys.platform == "darwin":
     extra_compile_args.append("--std=c++17")
     extra_compile_args.append("--stdlib=libc++")
-    extra_compile_args.append("-mmacosx-version-min=10.9")
+    extra_compile_args.append("-mmacosx-version-min=11.0")
```

Comment:
10.14 should be enough but let's first test that on the conda-forge feedstock
```suggestion
    extra_compile_args.append("-mmacosx-version-min=10.14")
```